cf_remote/commands.py

```diff
@@ -679,6 +706,15 @@ def deploy(hubs, masterfiles):
     if os.path.isfile(masterfiles):
         return deploy_tarball(hubs, masterfiles)
 
+    if masterfiles.endswith(".tgz"):
```

Comment:
`.tar.gz` also supported when the argument is checked below.

Reply:
Thanks for the review: https://github.com/cfengine/cf-remote/pull/30